src/controllers/trips.js

```diff
@@ -0,0 +1,73 @@
+import db from '../database/models/index';
+import jwt from 'jsonwebtoken'
+const Comment = db['Comment']
+
+export const commentOnRequests = async(req,res, next) => {
+    try{
+   
+      const { comment } = req.body;
+
+          const createComment = await Comment.create({
+            userId: req.user.id,
+            tripRequestId: req.params.tripRequestId,
+            comment,
+          });
+
+          if(createComment){
+          return res.status(201).json({message: 'comment successfully added'})
+        }
+    } 
+    catch(error){
+        return res.status(500).json({
+            error: error.message,
+            })
+        }
+}
+
+export const getComments = async(req, res) => {
+
+    try {
+         const tripRequestId = req.params.tripRequestId
+         const comments = await Comment.findAll({ where: { tripRequestId: tripRequestId } })
+
+    if (comments) {
+        return res.status(200).json({message: 'Comments fetched successfully', travelRequestId: tripRequestId, comments: comments})
+    }
+    } catch (error) {
+        return res.status(500).json({
+            error: error.message,
+            })
+        }
+    }
+
+export const deleteComment = async(req, res) => {
+    const authHeader = req.get('Authorization');
+    if (!authHeader) {
+        return res.status(401).json({message: 'Not authenticated'})
+    }
+
+    const token = authHeader.split(' ')[1];
+
+    if (!token) {
+        return res.status(401).json({message: 'Not authenticated'})
+    }
+
+    const userId = jwt.verify(token, process.env.JWT_SECRET).id;
+
+    const commentId  = req.params.id;
+    const comment = await Comment.findByPk(commentId);
+    console.log(comment)
+
+    if (comment.userId !== userId) {
```

Comment:
maybe you might want to allow also a super admin some right to delete comments

Reply:
> maybe you might want to allow also a super admin some right to delete comments

Done too;
Thanks!